src/main/java/racingcar/Car.java

```diff
@@ -4,13 +4,24 @@
 import java.util.List;
 
 public class Car {
-    private final static int GO_VALUE = 4;
+    private static final int GO_VALUE = 4;
+    private final String name;
```

Comment:
```suggestion
    private static final int GO_VALUE = 4;
    
    private final String name;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다 :)